lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -100,12 +98,12 @@ public function execute(
         InputInterface $input,
```

Comment:
Can we also change the visibility from "public" to "protected" on the Command class to match the declaration with Symfony/Console ?
If you agree, I may open a PR

Reply:
Sure! Did not know was protected